### ISSUE TITLE
Must we wait for ssh to stabilize?

### ISSUE THREAD

**Issue by tzachshabtay** [tanishq-dubey/harbormaster#1]
https://github.com/tanishq-dubey/harbormaster/blob/ab64fe0446654bc47f415365a9b3910fb48811a2/harbormaster.py#L38

What are we waiting for exactly here, and isn't there a way to not make it time based?

**Comment by tanishq-dubey** [comment]
Agreed that this does not have to be time based. The ideal method to instantiate the docker client in a try loop:
``` python
connected = False
while not connected:
    try:
        dClient = docker.DockerClient(base_url=f'tcp://localhost:{args.p}')
        connected = True
    except:
        print("not ready...")
        time.sleep(1) # <- need to wait a little to establish the initial SSH connection
```

With proper exception handling etc

Edit: The reasoning for doing the simple 5 second wait was to just get a v0.1 out the door.

**Comment by tanishq-dubey** [comment]
Fixed in #2
